test/fixtures/correct.js

```diff
@@ -233,13 +233,10 @@ noop(import1);
 noop(import3);
 noop(import4);
 
-// `sorting/sort-object-props`.
-const sortObjectProps1 = 'foo';
-const sortObjectProps2 = 'bar';
-
+// `sort-keys`.
 const sortObjectProps = {
-  [`${sortObjectProps1}`]: 'foo',
-  [`${sortObjectProps2}`]: 'bar'
+  var9: 'bar',
+  var10: 'foo'
```

Comment:
WDYT about adding a `var1` as well, to test the natural order? Otherwise, both tests could pass with a non-natural descending order.